stingray/lightcurve.py

```diff
@@ -623,7 +728,8 @@ def make_lightcurve(toa, dt, tseg=None, tstart=None, gti=None, mjdref=0,
             counts, histbins = np.histogram(toa[good], bins=histbins)
             time = histbins[:-1] + 0.5 * dt
 
-        return Lightcurve(time, counts, gti=gti, mjdref=mjdref, dt=dt)
+        return Lightcurve(time, counts, gti=gti, mjdref=mjdref, dt=dt,
+                          skip_checks=True, precise_poisson_err=False)
```

Comment:
I think this implies that by default, when creating a light curve from Poisson events, we're not ever generating any errors for it. I think this static method at the very least should have a `generate_errors` keyword or something? 

If we make a light curve out of events, we *know* that it ought to be Poissonian, so we know exactly which uncertainties to compute. I think by default, it should generate errors, to be honest.

Reply:
When the user uses `lc.count_err`, the errors get calculated. I don't see a case where the user is left uncertainty-less

Reply:
Oh right! The lazy evaluation is taking some getting used to. :D 

Is it maybe worth specifying explicitly `err_dist = "poisson"` here? Because it's not inconceivable that we might change the default behaviour of `err_dist` in the future, and that might lead to very obscure bugs in this case.

Reply:
Yep, that's true. Done